New guns, calibres, and conversion kits, fixed PR.

DoctorBoomstick opened the pull request:
#### Summary
Content "Migrates the changes from PR #64743. Adds 24 caliber conversion kits for various weapons, 13 new guns, 3 new calibres, and 3 professions, and migrates over the abandoned changes left behind in PR #63074."
#### Purpose of change
Following some rebase and merge errors, I had to close #64743, as it had become what could be generously described as a crap sandwich of errors and conflicts. This PR simply rescues and migrates over the files left behind in that PR. For the sake of ease of reference, the PR overview has been copied over.
Within the limits of real-world firearms’ use, caliber conversion kits for rifles (most notably the AR-15 platform) are rather prolific, with a number of such kits being in production and serving shooters within the remits of the roles they’re engineered for. This being said, such retool kits have been, for one reason or another, very thin on the ground within CDDA, with, at present, only a single .300 AAC Blackout kit existing in the whole game. This has led to guns like the abovementioned AR-15 rifles, as well as other arms like Glock pistols, not possessing the types of caliber swapping capabilities that their modular designs are prominently known for.
Personally, I believe that an effective implementation of such a mechanic could provide interesting tactical opportunities, as is the case with the Hub HWP. Apart from that, it could also assist in helping to elevate some firearms above their peers as far as individual utility goes and help them stand out from the current hordes of metal pue pue sticks that dump archaically named bullets into zombies.
Unrelated to caliber conversion kits, in PR #63074 I started to rework firearm damage modifiers in order to bring them up to speed with the currently established formula for determining such statistics. Due to various merge errors, rebase mistakes, and a large amount of IQ/coffee deprivation on my behalf, the entire PR got somewhat imploded following the release of 0.G, prompting me to shut it down.
#### Describe the solution
With respect to the caliber conversion kits, I have based their function on the current implementation of the HWP attachment system. Fundamentally, this involves having the base firearm itself simply be an empty frame, lacking any defined ammunition type, range value, damage amount, etc. Such values are only introduced with various removable and installable conversion kits that provide the weapons with their gun-related functionality.
As such, most of the AR-platform rifles, for example, have had their numbers tweaked to represent solitary lower receivers, with the differences made up by the installable conversion kits, which act as detachable upper receivers. All guns that possess this functionality are guaranteed to come with such a kit already installed by default, with these empty frames never spawning game-side. In addition, kits with differing barrel lengths have also been included, ranging from standard 16-inch assemblies to compact 7.5-inch CQB versions. These options allow players to indulge themselves in a very questionable game of dress-up with their guns by being able to retool their weapons according to their preferences and situations, like some form of metal Barbie doll.
A selection of screen shots in which this functionality is demonstrated is included here.
![Empty AR-15 Fraim](https://user-images.githubusercontent.com/102766473/230717383-da6235f2-911e-4195-a8e9-57e48d01a05f.png)
![7 62x39mm M4a1](https://user-images.githubusercontent.com/102766473/230717388-71916a43-a356-47d2-a641-6f79af3fe30e.png)
![450 Bushmaster H K 416](https://user-images.githubusercontent.com/102766473/230717390-8fd4bd8c-4637-46e1-9a66-ee446f34c560.png)
![50Beowulf MK 18](https://user-images.githubusercontent.com/102766473/230717399-43fa9fa5-ab5e-4a1d-a94c-497eeab11726.png)
#### Describe alternatives you've considered
I considered extending this PR further and adding a number of pistol-caliber and handgun-related conversions to the complement already present. I decided against this as pistol-related conversions would likely demand a similar amount of work and content to that already presented here, and I wished to try and keep this PR manageable in its size.
I also wished to add the 338 Norma Magnum as a caliber but opted to leave it for another day for the same reasons outlined above.
#### Testing
I spawned into an active game and observed that no errors manifested themselves. Following this, I systematically spawned a number of convertible guns and spent some time attaching and detaching several conversion kits from them, firing different calibres through the weapons, and generally trying to see where any incongruities might arise.
#### Additional context
What follows is a list of internet resources and links that I consulted for information used in the creation of this PR:
[Ruger® Hawkeye® Guide Gun Bolt-Action Rifle Model 47118](https://www.ruger.com/products/guideGun/specSheets/47118.html)
[Ruger Precision® Rifle Bolt-Action Rifle Model 18081](https://www.ruger.com/products/precisionRifle/specSheets/18081.html)
[Ruger Precision® Rifle .300 Win Mag / .300 PRC 5-Round Magazine (shopruger.com)](https://shopruger.com/Ruger-Precision-Rifle-300-Win-Mag-_-300-PRC-5-Round-Magazine/productinfo/90682/)
[CZ BREN 2 Ms CARBINE 223 REM (czub.cz)](https://www.czub.cz/en/firearms-and-products-product/cz-bren-2ms-carbine-223-rem)
[CZ BREN 2 Ms 7,62x39 11" (czub.cz)](https://www.czub.cz/en/firearms-and-products-product/cz-bren-2ms-7-62x39mm)
[Review: CZ Bren 2 Ms | An Official Journal Of The NRA (americanrifleman.org)](https://www.americanrifleman.org/content/review-cz-bren-2-ms/)
[Barrett MRAD .338 Lapua Bolt Action Rifle | Flat Dark Earth | Palmetto State Armory](https://palmettostatearmory.com/barrett-mrad-338-lapua-bolt-action-rifle-flat-dark-earth-14376.html)
https://www.savagearms.com/content?p=firearms&a=product_summary&s=22448
https://www.mcmillanfirearms.com/sacitem.aspx?item=mcmtac338lapua
[Accuracy International - AXMC Series Rifle – Sporting Services](https://www.sportingservices.co.uk/products/ax-series-rifle)
https://www.czub.cz/en/firearms-and-products-product/cz-600-trail
https://www.ruger.com/products/americanRifleRanch/specSheets/16950.html
https://thegunzone.com/best-ar-15-calibers-conversion-kits/
https://www.pewpewtactical.com/best-ar-15-calibers-conversion-kits/#toc5
https://www.americanrifleman.org/content/first-look-cmmg-5-7x28-mm-ar-conversion/
https://www.57center.com/
https://www.bearcreekarsenal.com/300-blackout-16-parkerized-heavy-1-8-twist-pistol-15-hanguard-rear-forged-nrail.html
https://underwoodammo.com/12-7x42-designed-for-use-in-50-beowulf-firearms-420gr-xtreme-penetrator-solid-monolithic-hunting-self-defense-ammo/
[12.7x42 AR-15 Heavy Barrel | High Accuracy (bearcreekarsenal.com)](https://www.bearcreekarsenal.com/12-7x42-18-parkerized-heavy-barrel-1-20-twist-mid-length-gas-system-barrel.html)
https://www.bearcreekarsenal.com/12-7-x-42-16-parkerized-hb-1-20-twist-carbine.html
https://www.bearcreekarsenal.com/bca-ar-15-barrel-10-5-4150-black-nitrided-steel-heavy-barrel-12-7x42-pistol-length-gas-system-w-1-20-twist.html
https://www.bearcreekarsenal.com/5-56-nato-16-parkerized-m4-mid-length-1-7-twist-15-handguard-upper-nrail-spiral-flash.html
https://www.bearcreekarsenal.com/5-56-nato-10-5-parkerized-m4-1-7-twist-carbine-standard-handguard-upper.html
https://deserttech.com/conversion-kits.php
https://www.cheaperthandirt.com/ammunition/rifle-ammo/.450-bushmaster/
https://www.sellier-bellot.cz/en/products/rifle-ammunition/rifle-ammunition-target-match/detail/475/
https://www.nammo.com/product/our-products/ammunition/small-caliber-ammunition/8-6mm-series-338/8-6-mm-x-70-338-lapua-magnum-armor-piercing/
[Tactical ammunition for MIL / LE | AP and API projectiles - Lapua](https://www.lapua.com/purpose/tactical/?wpf_filter_cat_list_1=131&wpf_filter_caliber=338-lapua-mag&all_products_filtering=1)
[AXMC (accuracyinternational.com)](https://accuracyinternational.com/axmc.html)
[Search results for: '.450' (bearcreekarsenal.com)](https://www.bearcreekarsenal.com/catalogsearch/result/?q=.450)
https://www.primaryarms.com/radical-firearms-762x39mm-barreled-upper-16in-primary-arms-exclusive
https://www.righttobear.com/7-62-39-ar-magazines/

DoctorBoomstick committed:
adds the files in the items/ammo folder.

DoctorBoomstick committed:
adds the files in the items/armour folder.

DoctorBoomstick committed:
adds the files in the items/book folder.

DoctorBoomstick committed:
adds the files in the items/container folder.

DoctorBoomstick committed:
adds the files in the items/generic folder.

DoctorBoomstick committed:
adds the files in the items/gun folder.

DoctorBoomstick committed:
adds the files in the items/gunmod folder.

DoctorBoomstick committed:
adds the files in the items/magazine folder.

DoctorBoomstick committed:
adds asorted files in the items folder.

DoctorBoomstick committed:
adds the conversions folder in the itemgroups/Weapons_Mods_Ammo folder.

DoctorBoomstick committed:
adds files in the itemgroups/Weapons_Mods_Ammo/arsenal folder.

DoctorBoomstick committed:
adds the files in the itemgroups/Weapons_Mods_Ammo folder.

DoctorBoomstick committed:
adds files in the itemgroups folder.

DoctorBoomstick committed:
adds files in the itemgroups/SUS folder.

DoctorBoomstick committed:
adds files in the uncraft/ammo folder.

DoctorBoomstick committed:
adds files in the requirements folder.

DoctorBoomstick committed:
adds files in the recipes folder.

DoctorBoomstick committed:
adds files in the json folder.

DoctorBoomstick committed:
adds files in the mods/Generic_Guns folder.

DoctorBoomstick committed:
last tweeks.

TheShadowFerret commented:
>I considered extending this PR further and adding a number of pistol-caliber and handgun-related conversions to the complement already present. I decided against this as pistol-related conversions would likely demand a similar amount of work and content to that already presented here, and I wished to try and keep this PR manageable in its size.

👀 I hope this will be more than just "Glock 22 & 31 can switch between .357 SIG & .40 S&W"

DoctorBoomstick commented:
> > I considered extending this PR further and adding a number of pistol-caliber and handgun-related conversions to the complement already present. I decided against this as pistol-related conversions would likely demand a similar amount of work and content to that already presented here, and I wished to try and keep this PR manageable in its size.
> 
> 👀 I hope this will be more than just "Glock 22 & 31 are made one item with two conversion kits" when it does come

I need to do some further research on the matter to see what conversions exist specifically for pistols, but it’ll be more. all the PCC conversions for AR-15s need to be acounted for, for one, and there are a couple other things which I do know exist and would like to add in.

TheShadowFerret commented on data/json/items/gun/223.json:
```diff
@@ -391,24 +474,108 @@
         "id": "m4_cqbr",
         "name": { "str": "Mk 18 CQBR" },
-        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations."
+        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations.",
```
I don't think this needs to exist as a seperate item anymore, given it can be assembled from the regular M4

DoctorBoomstick replied:
> I don't think this needs to exist as a seperate item anymore, given it can be assembled from the regular M4 & a CQB upper.

This is true, but given that gun mods can’t append the name of a weapon, I’d personally feel a bit iffy about up and removing it, as I’d like there to be some form of name difference between a regular M4 and the Mk 18 variant.

TheShadowFerret commented on data/json/items/magazine/308.json:
```diff
@@ -300,5 +300,22 @@
+    "color": "light_gray",
+    "ammo_type": [ "308" ],
+    "flags": [ "MAG_COMPACT" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "308": 10 } } ]
```
```suggestion
    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "338lapua": 10, "300": 10 } } ]
```
To match item description

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
fixes a typo in the disc for the AI .308 10-round magazine.

LordZanos commented on data/json/items/ammo/3006.json:
```diff
@@ -17,7 +17,7 @@
     "casing": "3006_casing",
     "range": 61,
-    "damage": { "damage_type": "bullet", "amount": 62, "armor_penetration": 10 },
+    "damage": { "damage_type": "bullet", "amount": 49, "armor_penetration": 10 },
```
Changes to 30-06 should probably be duplicated on 12.3ln, which supposedly has identical ballistic performance.

LordZanos commented on data/json/items/ammo/308.json:
```diff
@@ -17,7 +17,7 @@
     "casing": "308_casing",
     "range": 65,
-    "damage": { "damage_type": "bullet", "amount": 60, "armor_penetration": 2 },
+    "damage": { "damage_type": "bullet", "amount": 54, "armor_penetration": 2 },
```
I don't see corresponding changes to the 308 guns in this PR. They were in the (now closed) rebalance PR. Not sure if it's intentional that the edits to the weapons aren't here?

DoctorBoomstick replied:
> I don't see corresponding changes to the 308 guns in this PR. They were in the (now closed) rebalance PR. Not sure if it's intentional that the edits to the weapons aren't here?

You know, I suspected I forgot to port something over when creating this PR. Thanks for pointing it out. On the subject of the two 12.3mm LN rifles, my singular question is what should their barrel lengths be? I suppose I can look at their respective lengths and try to make a guestimation based on that.

DoctorBoomstick committed:
general damage tweeks.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick commented:
@LordZanos , I’ve updated and added the damage modifiers that you pointed out. Surprisingly enough, 12.3ln has a different damage amount and related modifier than .30-06, so go figure. As a side note, I took the liberty to also update damage modifiers for .300 Win Mag rifles, one of which I added in this PR, and 8x40mm guns, which I have removed the never jams flag from in this pr. A list of all the modifier changes can be observed within this spreadsheet.
[Firearm Damage Modifiers2.xlsx](https://github.com/CleverRaven/Cataclysm-DDA/files/11220904/Firearm.Damage.Modifiers2.xlsx)

LordZanos commented:
> @LordZanos , I’ve updated and added the damage modifiers that you pointed out. Surprisingly enough, 12.3ln has a different damage amount and related modifier than .30-06, so go figure. As a side note, I took the liberty to also update damage modifiers for .300 Win Mag rifles, one of which I added in this PR, and 8x40mm guns, which I have removed the never jams flag from in this pr. A list of all the modifier changes can be observed within this spreadsheet. [Firearm Damage Modifiers2.xlsx](https://github.com/CleverRaven/Cataclysm-DDA/files/11220904/Firearm.Damage.Modifiers2.xlsx)

You might want to ask rivet about changes to rivtech guns, they didn't follow the same pattern even before. Not sure why.

DoctorBoomstick commented on data/json/items/ammo/exodii.json:
```diff
@@ -5,6 +5,7 @@
     "description": "In some alternate universe, the 12.3ln cartridge was introduced in Romania in the wake of the second Carpathian conflict.  The PA md. 71 rifle using this ammunition rapidly gained popularity in the Eastern Union, and from there, the world.  Due to this, the 12.3ln rapidly became the standard combat round in the Eurasian sphere, making it easy for the Exodii to stockpile it in  ...
-    "ammo_type": "123ln"
+    "ammo_type": "123ln",
+    "damage": { "damage_type": "bullet", "amount": 57, "armor_penetration": 10 }
```
In the game balance docs, there’s calculated damage listings for all the possible cartridges, as well as formulae for deriving damage modifiers based upon barrel lengths. 12.3ln has a listed base damage of 57, and .30-06 has a listing of 49, so I had to update the round’s json to reflect this. Same goes with the calculated damage values for 8x40mm.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

fixes merge erros.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

merges files.

DoctorBoomstick committed:
fixes merge errors.

Zireael07 commented on data/json/items/ammo/exodii.json:
```diff
@@ -6,6 +6,35 @@
     "name": { "str": "12.3ln round" },
     "description": "In some alternate universe, the 12.3ln cartridge was introduced in Romania in the wake of the second Carpathian conflict.  The PA md. 71 rifle using this ammunition rapidly gained popularity in the Eastern Union, and from there, the world.  Due to this, the 12.3ln rapidly became the standard combat round in the Eurasian sphere, making it easy for the Exodii to stockpile it in  ...
     "ammo_type": "123ln",
+<<<<<<< HEAD
```
Got a bad merge here

DoctorBoomstick replied:
should be fixed now.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

merges files.

DoctorBoomstick committed:
fixes merge errors.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

fixes merge errors.

DoctorBoomstick committed:
starts fixing bugs.

DoctorBoomstick committed:
fixes bugs.

DoctorBoomstick committed:
hooks up the ammo with the new barrel length system. still needs to add the barrel lengths to the guns themselves.

DoctorBoomstick committed:
adds the barrel lengths to the .223 guns.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

fixes merge errors.

GuardianDll commented on data/json/items/ammo/450bushmaster.json:
```diff
@@ -0,0 +1,128 @@
+    "name": { "str_sp": ".450 Bushmaster FTX" },
+    "description": ".450 Bushmaster ammunition, loaded with a plastic-capped 250-grain Flex Tip bullet.  Born from discontent with the .223 Remington’s weak stopping potential, the .450 Bushmaster round was crafted in line with the so-called 'Thumper' concept: an envisioned need for a large-bore, semi-automatic sporting cartridge capable of downing game in one shot.  Along with its elder sibling, ...
+    "weight": "16 g",
+    "volume": "145 ml",
```
```suggestion
    "volume": "145 ml",
    "longest_side": "58 mm",
```

GuardianDll commented on data/json/items/ammo/50_beowulf.json:
```diff
@@ -0,0 +1,146 @@
+    "name": { "str_sp": ".50 Beowulf XTP" },
+    "description": ".50-caliber Beowulf ammunition, loaded with a copper-jacketed 350-grain eXtreme Terminal Performance bullet.  Developed in 2001 as a powerful short- to medium-range chambering for AR-15 style rifles, this stout cartridge had gained some popularity as a hunting round before the Cataclysm.  This particular version of the cartridge is coated in a layer of polymer, encouraging com ...
+    "weight": "22 g",
+    "volume": "231 ml",
```
```suggestion
    "volume": "231 ml",
    "longest_side": "54 mm",
```

GuardianDll reviewed (commented):
somewhat miss this two initially

GuardianDll commented on data/json/items/book/rifle.json:
```diff
@@ -60,7 +60,7 @@
       {
         "id": "manual_rifle_1",
-        "name": { "str": "SAS and Elite Forces Guide Sniper", "str_pl": "copies of SAS and Elite Forces Guide Sniper" },
+        "name": { "str": "SAS and Elite Sniper Forces Guide", "str_pl": "copies of SAS and Elite Sniper Forces Guide" },
```
It is not a name of a book in the comment, isnt it? But it sound silly in this order, i agree

DoctorBoomstick replied:
Apologies about the book variant title, but I neglected to double check it with the actual source and changed it as it sounded somewhat silly. Would you rather I change it back to its original form?

GuardianDll commented on data/json/items/book/rifle.json:
```diff
@@ -119,7 +119,8 @@
         },
-        "description": "Official U.S. Army manual which provides guidance for training with the 5.56-millimeter M16-series rifle and M4 carbine.",
+        "description": "An official U.S. Army manual that provides guidance for the planning and execution of training with the 5.56mm M16-series rifle and M4 carbine.",
+        "proficiencies": [ { "proficiency": "prof_gunsmithing_basic", "time_factor": 0.8, "fail_factor": 0.8 } ],
```
I doubt our variant system is capable to make it work

DoctorBoomstick committed:
Update data/json/items/ammo/338lapua.json

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>

DoctorBoomstick committed:
Update data/json/items/ammo/450bushmaster.json

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>

DoctorBoomstick committed:
Update data/json/items/ammo/50_beowulf.json

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>

DoctorBoomstick committed:
Update data/json/items/ammo/338lapua.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

DoctorBoomstick committed:
Adds a 20-inch conversion .223 kit for AR-15 rifles, representing those found on M16s. also adds in all the barrel-lengths for weapons in .300, .308, .30-06, and 12.3ln.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Commits suggested changes.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

bombasticSlacks commented on data/json/items/ammo/223.json:
```diff
@@ -18,7 +18,37 @@
+        { "barrel_length": "186 mm", "amount": 32 },
+        { "barrel_length": "206 mm", "amount": 33 },
+        { "barrel_length": "229 mm", "amount": 34 },
+        { "barrel_length": "249 mm", "amount": 35 }
```
This feels like not enough entries for 223

bombasticSlacks commented on data/json/items/generic/casing.json:
```diff
@@ -30,7 +31,6 @@
   },
   {
     "id": "22_casing_new",
-    "looks_like": "9mm_casing",
```
Why remove looks like?

DoctorBoomstick committed:
fixes the, "last," bug. I hope. please God.

bombasticSlacks commented on data/json/items/gun/shot.json:
```diff
@@ -52,7 +52,7 @@
     "durability": 8,
     "modes": [ [ "DEFAULT", "burst", 6 ] ],
-    "energy_drain": "1 kJ",
+    "ups_charges": 1,
```
This seems to be an oversight I believe this was changed

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Does a broad stroke to get gun migrations underway. Hope this doesn’t break… everything
.

DoctorBoomstick committed:
fixes pointed out bugges.

DoctorBoomstick committed:
actualy fixes the shotgun bug.

DoctorBoomstick committed:
lints, lints, lints, and lints some more.

DoctorBoomstick committed:
Fixes the solitary erroneous file.

DoctorBoomstick committed:
fixes more bugs.

DoctorBoomstick committed:
retrigger checks

DoctorBoomstick committed:
tweeks and bug fixes, makes the long version of the ar15 upper findable on guns and usable in crafts/ weapon mods.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Adds/ fixes gun entries in generic guns.

DoctorBoomstick committed:
adds the last gun to generic guns.

DoctorBoomstick committed:
look at that, more bug fixes.

DoctorBoomstick committed:
adds more values for .223 barrel lengths and removes ranged damage entries from the conversions. they were causing errors, and the new barrel length system should make them unneeded anyhow.

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Adds name plural forms for the conversion kits which need them.

DoctorBoomstick committed:
Gives the prepped patriot a leg bag for storage.

DoctorBoomstick committed:
adds no turret flags to all the guns which need it.

DoctorBoomstick committed:
fixes a tiny bug.

DoctorBoomstick committed:
the M231 wasn't lacking an ammo entry. No. I didn't forget that. this commit doesn't add that entry.

DoctorBoomstick committed:
retrigger checks

DoctorBoomstick committed:
retrigger checks

bombasticSlacks commented on data/json/items/gun/338lapua.json:
```diff
@@ -168,6 +168,8 @@
       [ "stock", 1 ]
     ],
+    "flags": [ "NO_TURRET" ],
+    "//2": "This should be removed once vehicle turrets are capable of mounting guns with mods attached. As it stands, trying to mount this style of weapon would require the player to remove the conversion kit, thus rendering it unusable anyhow.",
```
```suggestion
    "//3": "This should be removed once vehicle turrets are capable of mounting guns with mods attached. As it stands, trying to mount this style of weapon would require the player to remove the conversion kit, thus rendering it unusable anyhow.",
```

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

bombasticSlacks commented on data/json/items/gun/3006.json:
```diff
@@ -90,12 +91,14 @@
     "material": [ "steel", "wood" ],
     "color": "brown",
     "ammo": [ "3006" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": 8 },
```
Should this have both barrel and damage?

DoctorBoomstick replied:
Oops, forgot to axe that when I was doing the barrel-length updates. thanks for pointing it out.

TheShadowFerret commented on data/json/items/magazine/762.json:
```diff
@@ -136,5 +136,108 @@
+    "looks_like": "stanag30",
+    "type": "MAGAZINE",
+    "name": { "str": "Bren 2 10-round magazine" },
+    "description": "A proprietary polymer box magazine, capable of containing 10 7.62x39mm rounds and compatible with the Check Bren 2 carbine.  Unlike conventional 7.62x39mm magazines, this one is inserted straight up into the magazine well rather than rocked in, in order to facilitate a bolt-hold open mechanism.",
```
```suggestion
    "description": "A proprietary polymer box magazine, capable of containing 10 7.62x39mm rounds and compatible with the Czech Bren 2 carbine.  Unlike conventional 7.62x39mm magazines, this one is inserted straight up into the magazine well rather than rocked in, in order to facilitate a bolt-hold open mechanism.",
```

DoctorBoomstick committed:
Update data/json/items/magazine/762.json

Co-authored-by: TheShadowFerret <99621099+TheShadowFerret@users.noreply.github.com>

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Removes the redundant ranged damage entry.

DoctorBoomstick committed:
Merge branch 'more_boomsticks_retooled' of https://github.com/DoctorBoomstick/Cataclysm-DDA into more_boomsticks_retooled

bombasticSlacks committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

fixes merge errors.

DoctorBoomstick committed:
fixes blacklist errors and adds the post-2000 guns (i.e. all the guns) to the classic zombies blacklist.

DoctorBoomstick committed:
Merge branch 'more_boomsticks_retooled' of https://github.com/DoctorBoomstick/Cataclysm-DDA into more_boomsticks_retooled

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

DoctorBoomstick committed:
Merge branch 'master' into more_boomsticks_retooled

bombasticSlacks commented:
In the future, being able to describe gun groups that are allowed for mods, or have variants spawn with mods to clean up some ID bloat would be nice, but no need to hold this up further.
